package.json

```diff
@@ -4,11 +4,13 @@
   "repository": "https://github.com/dimagi/commcare-hq",
   "description": "CommcareHQ Javascript tests",
   "dependencies": {
+    "biginteger": "^1.0.3",
```

Comment:
Given your last comment I may want to reconsider my stance on modifying js-xpath. Perhaps we could sit down together and I can show you where the trouble with biginteger and schemeNumber are coming from and decide whether it's easily patchable within js-xapth